packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -494,8 +491,7 @@ function registerInteractionListener(
   const registerInteractionTransaction = (): void => {
     const op = 'ui.action.click';
 
-    const activeSpan = getActiveSpan();
-    const rootSpan = activeSpan && getRootSpan(activeSpan);
+    const rootSpan = getActiveIdleSpan(client);
```

Comment:
m/h: I'm a bit concerned about this as we don't check the scope anymore for the active span. Which means we diverge here from other places where we do something similar. Also, paired with my other comment, we'd need to ensure that we unset the span on the client whenever the span ends.

Reply:
so I adjusted the idle spans to be unset when the idle (route) span ends. Then, I _think_ this should be OK?